### ISSUE TITLE
get_raid_level fails when no other forms have yet been used ingame

### ISSUE THREAD

**Issue by Artanicus** [florianbecker/PokemonRaidBot#172]
Background info: https://github.com/Map-A-Droid/MAD/issues/989
TL;DR; While the form_id will exist for a long time before a mon has more than one form it may not actually be used ingame until another form is published and they need to be deduped.

We however assume that the form_id will be "correct" when receiving raids over webhook so when MAD sees a Heatran raid it will happily send a form of 0 but we expect 1862.

The solution is on a zero form translate it to the normal form since that should be a safe bet. When/if another form is actually published we shouldn't receive a zero anymore and then it will keep on working correctly both for the old and the new form.

**Comment by Artanicus** [comment]
@Ninjasoturi You had a preliminary fix for this, is it ready for a PR?

**Comment by Ninjasoturi** [comment]
@Artanicus Yep, a suggestion for a fix  in PR #176

**Comment by Artanicus** [comment]
Fixed by #176
